spec/models/manageiq/providers/amazon/storage_manager/ebs/stubbed_refresher_spec.rb

```diff
@@ -53,15 +53,15 @@
         it "2 refreshes, first creates all entities, second deletes all entitites" do
           @data_scaling = 1
           2.times do
-            refresh_spec
+            refresh_spec(false)
```

Comment:
I'd go with two functions instead of this boolean:
* `refresh_spec_table_counts`
* `refresh_spec_specific_objects` (should assert both table counts and specific snapshot/volume)

Or at least rename the parameter `with_specific_tests` into `assert_counts_only`. I was wondering what specific tests are you referring to and could only understand when I read the comment here...